app.config.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
it's better to change this file to a template, and add app.config.json to .gitignore